pygraphviz/tests/test_readwrite.py

```diff
@@ -1,39 +1,34 @@
 import pygraphviz as pgv
-import os
-import tempfile
-import pathlib
 
 
-def test_readwrite():
+def test_readwrite(tmp_path):
     A = pgv.AGraph(name="test graph")
     A.add_path([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
-    the_file = tempfile.NamedTemporaryFile(delete=False)
-    fname = the_file.name
-    # Make sure it can be opened for writing again on Win32
```

Comment:
It's probably also worth keeping this comment, otherwise it'll be really hard to figure out why we're opening and closing a file prior to interacting with it!